src/main/resources/application.properties

```
@@ -1 +1,6 @@
-amazon.sqs.end-point.poll-queue= ${SQS_ENDPOINT_POLL_QUEUE}
\ No newline at end of file
+amazon.sqs.end-point.poll-queue= ${SQS_ENDPOINT_POLL_QUEUE}
+
+api.operations.baseUrl=${API_OPERATIONS_BASEURL}
+
+api.eservice.basePath=${API_ESERVICE_BASEPATH}
```

as per its use, this is part of the `operations` configuration. 
It should be something like
```suggestion
api.operations.eservice.basePath=${API_ESERVICE_BASEPATH}
```

or  if you prefer, you can group the configurations like

```
api.operations {
  baseUrl = ${API_OPERATIONS_BASEURL}
  eservices.basePath = ${API_ESERVICE_BASEPATH}
}
```

I would also change the env var name to reflect this correlation
`API_ESERVICE_BASEPATH` -> `API_OPERATIONS_ESERVICE_BASEPATH`